src/settings/__tests__/settingsReducer-test.js

```diff
@@ -24,17 +24,17 @@ describe('settingsReducer', () => {
         onlineNotification: false,
         streamNotification: false,
       });
-
-      const action = eg.mkActionRegisterComplete({
-        enable_offline_push_notifications: true,
-        enable_online_push_notifications: true,
-        enable_stream_push_notifications: true,
-        user_settings: undefined,
-      });
-
-      const actualState = settingsReducer(prevState, action);
-
-      expect(actualState).toMatchObject({
+      expect(
+        settingsReducer(
+          prevState,
+          eg.mkActionRegisterComplete({
+            enable_offline_push_notifications: true,
+            enable_online_push_notifications: true,
+            enable_stream_push_notifications: true,
+            user_settings: undefined,
+          }),
+        ),
+      ).toMatchObject({
```

Comment:
Yep, agreed.

I think in a lot of these reducer tests, pulling the action out has this effect, because the action is usually long and the previous state is usually already a local variable. So one approach on those trade-offs would be to consistently pull the action out.